core/services/eth/log_broadcaster_test.go

```diff
@@ -160,20 +160,17 @@ func TestLogBroadcaster_ResubscribesOnAddOrRemoveContract(t *testing.T) {
 }
 
 type simpleLogListner struct {
-	handler func(lb ethsvc.LogBroadcast, err error)
-	id      models.ID
+	handler    func(lb ethsvc.LogBroadcast, err error)
+	consumerID *models.ID
```

Comment:
Why is it a pointer?

Reply:
`models.ID` is a pointer all over the app. Just cheaper I guess.